client/package.json

```diff
@@ -2,8 +2,13 @@
 	"name": "client",
 	"version": "1.0.0",
 	"dependencies": {
+		"@fortawesome/fontawesome-svg-core": "^1.2.17",
+		"@fortawesome/free-solid-svg-icons": "^5.8.1",
+		"@fortawesome/react-fontawesome": "^0.1.4",
 		"axios": "^0.18.0",
+		"i": "^0.3.6",
```

Comment:
what is this?

Reply:
This I don't know; I don't remember adding it